eo-runtime/src/main/eo/org/eolang/tuple.eo

```diff
@@ -57,10 +57,16 @@
         index.lt 0
         index.gte len
       error "Given index is out of tuple bounds"
-      if.
-        index.lt (len.plus -1)
-        ^.head.at index
-        ^.tail
+      at-without-checks index
+
+    # Takes element from the tuple without index validation.
+    # Here i must be an object that can be comparable with int using lt attribute.
```

Comment:
@levBagryansky let's quote "i", "int" and "lt" with ` brackets so it'll be possible to render comments in markdown (in the future)